api/inventorypb/agents.proto

```diff
@@ -96,6 +96,8 @@ message MySQLdExporter {
   string service_id = 4;
   // MySQL username for scraping metrics.
   string username = 5;
+  // Custom password for exporter endpoint /metrics.
+  string agent_password = 18;
```

Comment:
is it requested by PO? I'm not sure that it's a good idea to share credentials.

Reply:
@BupycHuk Yes, it is in acceptance criteria here:
https://jira.percona.com/browse/PMM-7570

Reply:
Othervise I can add flag and we can wait for Roma to verify it.

Reply:
Maybe it's better to merge without this field and prepare another PR in case @rnovikovP insist on it.
I would suggest to add boolean field instead of string one to differentiate where we use id and where we use custom password.

Reply:
Exposing is reverted on all PRs.